packages/core/package.json

```diff
@@ -89,6 +80,7 @@
   "homepage": "https://github.com/mondaycom/vibe#readme",
   "dependencies": {
     "@popperjs/core": "2.11.6",
+    "@vibe/icons": "^0.0.4",
```

Comment:
also should not be a dependency in my opinion, but rather a dev dependency
it should only be shown on our Storybook without exporting it again through core

Reply:
oh never mind, forgot our components also consume it
I wonder how would this affect the bundle size then 👀